zetaclient/chains/bitcoin/observer/outbound.go

```diff
@@ -26,6 +26,8 @@ func (ob *Observer) GetTxID(nonce uint64) string {
 }
 
 // WatchOutbound watches Bitcoin chain for outgoing txs status
+// TODO(revamp): move ticker functions to a specific file
+// TODO(revamp): move into a separate package
```

Comment:
What's the name of the package and on which level?  Now we have parent package `bitcoin` and two sub packages of `observer` and `signer`

Reply:
TBD

Reply:
okay.